### DISCUSSION TITLE
How to pass DataTime parameter when query on GraphQL?

### DISCUSSION THREAD

**Discussion by 45592858** [prisma/prisma#4251] [Q&A] [answered]
# Write your query or mutation here
mutation {
  createOneMember(
    data: {
      No: 1
      name: "张小龙"
      gender: "男"
      birthDate: "2019-10-01"
      fatherName: "张大龙"
      fatherMobile: "13813813888"
      motherName: "李四"
      motherMobile: "13813813889"
    }
  ) {
    id
  }
}
---------------------------------------------------------
{
  "data": null,
  "errors": [
    {
      "message": "\nInvalid `prisma.member.create()` invocation:\n\n{\n  data: {\n    No: 1,\n    name: '张小龙',\n    gender: '男',\n    birthDate: '2019-10-01',\n               ~~~~~~~~~~~~\n    fatherName: '张大龙',\n    fatherMobile: '13813813888',\n    motherName: '李四',\n    motherMobile: '13813813889'\n  }\n}\n\nArgument birthDate: Got invalid value '2019-10-01' on prisma.createOneMember. Provided String, expected DateTime.\n\n",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "createOneMember"
      ]
    }
  ]
}
----------------------------------------------------
model Member {
  id           Int        @id @default(autoincrement())
  memberNo     String     @unique
  name         String
  gender       String
  birthDate    DateTime
  idCard       String?
  address      String?
  fatherName   String
  fatherMobile String
  motherName   String
  motherMobile String
  remark       String?
  createBy     Int?
  createAt     DateTime?  @createAt
  updateBy     Int?
  updateAt     DateTime?  @updateAt
}

**Comment by ryands17** [comment, marked as answer]
@45592858 you can pass DateTime directly as a string in the ISO format.

For e.g.

```graphql
mutation {
  createOneMember(
  data: {
    No: 1
    name: "张小龙"
    gender: "男"
    birthDate: "2019-10-01T00:00:00.000Z"
  }
) {
    id
  }
}
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
